main.go

```diff
@@ -86,8 +122,8 @@ func main() {
 	}
 
 	// Get list of DFO active contacts
-	dfoGetActiveContactsApiUrl := "https://api-de-na1.dev.niceincontact.com/dfo/3.0/contacts?status[]=new&status[]=resolved&status[]=escalated&status[]=pending&status[]=open"
-	dfoResponse := GetDfoActiveContactList(dfoGetActiveContactsApiUrl)
+	dfoActiveContactsApiUrl := "https://api-de-na1.dev.niceincontact.com/dfo/3.0/contacts?status[]=new&status[]=resolved&status[]=escalated&status[]=pending&status[]=open"
```

Comment:
With the change recommended above, this would become:

dfoActiveContactsApiUrl := fmt.printf("%s/contacts?status[]=new&status[]=resolved&status[]=escalated&status[]=pending&status[]=open", DfoBaseUri);